<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createserverdialog</class>
 <widget class="QDialog" name="createserverdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    /* --- حالت عادی دکمه --- */
    background-color: #555555; /* رنگ پس‌زمینه کمی روشن‌تر از کل صفحه */
    color: white;              /* رنگ متن */
    border: 1px solid #777777; /* یک حاشیه مشخص برای دکمه */
    padding: 8px 16px;         /* فاصله متن از لبه‌های دکمه */
    border-radius: 5px;        /* گرد کردن گوشه‌های دکمه */
    font-size: 14px;
}

QPushButton:hover {
    /* --- وقتی ماوس روی دکمه قرار می‌گیرد --- */
    background-color: #666666; /* پس‌زمینه کمی روشن‌تر می‌شود */
    border-color: #888888;
}

QPushButton:pressed {
    /* --- وقتی دکمه فشرده می‌شود --- */
    background-color: #444444; /* پس‌زمینه کمی تیره‌تر می‌شود */
    border-style: inset;       /* ایجاد حس فرورفتگی */
    padding-left: 18px;        /* حرکت جزئی متن برای حس کلیک شدن */
    padding-top: 10px;
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lblIP">
      <property name="text">
       <string>getting ip ...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="lblPort">
      <property name="text">
       <string>getting port...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listUsers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCloseParty">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>close party</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>createserverdialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>createserverdialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
